language/use-cases/sql-code-generation/sql_code_generation.ipynb

```diff
@@ -0,0 +1,1609 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #31.    print("{} user action records loaded to table {}".format(result.total_rows, TABLE_ID))
</pre><p>Is there a reason you're using <code>.format()</code> instead of an f-string? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/275/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>